notebooks/Batchelor.ipynb

```diff
@@ -0,0 +1,348 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0c08f601-4df0-44f4-b0e6-ad1a9d545f06",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dolfinx import *\n",
```

Comment:
Could you switch from using
```
from ___ import *
```
in this cell to importing individual things or importing them into a namespace.

e.g.:
```
import dolfinx as df
import dolfinx.fem.petsc
from mpi4py import MPI
```

Importing using `*` is dangerous because namespaces get combined and you're never quite sure which version of a function you're using.

Remember that after changing this cell you'll need to run the whole notebook and make changes to account for the new namespace prefixes.